ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node/Tracers.hs

```diff
@@ -74,6 +76,7 @@ data Tracers' remotePeer localPeer blk f = Tracers
   , gddTracer                     :: f (TraceGDDEvent remotePeer blk)
   , csjTracer                     :: f (TraceLabelPeer remotePeer (CSJumping.TraceEventCsj remotePeer blk))
   , dbfTracer                     :: f (CSJumping.TraceEventDbf remotePeer)
+  , kesAgentTracer                :: f KESAgentClientTrace
```

Comment:
This tracer is defined but unused

Reply:
Oh dear, yes... looks like I forgot to actually implement that part. I'll take a look ASAP.